src/mqttkat/client.clj

```diff
@@ -80,9 +80,9 @@
 
 (defn disconnect [client]
   (let [map (gen/generate (s/gen :mqtt/disconnect))
-        bufs (MqttDisconnect/encode map)]
+        bufs (MqttDisconnect/encode)]
```

Comment:
Can't we get rid of the map completely in this case? it isn't used later on anyway. And then we can get rid of the `let` as well.

Reply:
Ok, I will remove unused feature.

Reply:
I removed it.